resources/shaders/pp_bloom_composition.hlsl

```diff
@@ -47,7 +47,7 @@ void main_cs(int3 dispatch_thread_id : SV_DispatchThreadID)
 	float3 finalcolor = float3(0, 0, 0);
 	float bloom_intensity = 1.f;
 
-	if (enable_bloom > 0)
+	if (enable_bloom < 0)
```

Comment:
check with @DBouma

Reply:
Should be > 0.

Reply:
Did that for testing purposes, forgot to remove it, thanks.